wagtail/admin/auth.py

```diff
@@ -137,7 +138,7 @@ def user_has_any_page_permission(user):
 
     # At least one of the users groups has a GroupPagePermission.
     # The user can probably do something.
-    if GroupPagePermission.objects.filter(group__in=user.groups.all()).exists():
+    if bool(PagePermissionPolicy().get_cached_permissions_for_user(user)):
         return True
```

Comment:
Yep, I initially made it so that `user_has_any_permission` returns `True` if the `actions` is an empty list and the user has any `GroupPagePermission` (after the basic `is_superuser`/`is_active` checks, and reduced this function's body into `PagePermissionPolicy().user_has_any_permission(user, actions={})`.

However, then I realised this isn't consistent with the behaviour of the other permission policies (unless we want to update them) and `any()`, so I undid the changes.

I've updated this to check for the permissions we currently have in a2f4c1228fb7691bc68ff04b3b2e05b08a8d5202.